src/app/static/src/app/components/plots/bubble/BubblePlot.vue

```diff
@@ -393,17 +393,21 @@
                 const sizeIndicatorName = this.indicatorNameLookup[sizeIndicator];
                 const { format, scale, accuracy } = this.colorIndicator!;
                 const { format: formatS, scale: scaleS, accuracy: accuracyS } = this.sizeIndicator!;
-                const stringUpper_value = (upper_value || upper_value === 0) ? upper_value.toString() : "";
-                const stringSizeUpper = (sizeUpper || sizeUpper === 0) ? sizeUpper.toString() : "";
-                if(lower_value && sizeLower) {
+
+                const stringLower_value = lower_value ? lower_value.toString() : "";
```

Comment:
Again, this is going to cause problems for 0 values. I think the code as you had it before would be fine (if it was for both lower and upper values) if 0s were being returned from the utils methods - which they should be! (but aren't at the moment, I don't think)